kernel/src/syscall/mprotect.rs

```diff
@@ -22,9 +22,16 @@ pub fn sys_mprotect(addr: Vaddr, len: usize, perms: u64, ctx: &Context) -> Resul
     if len == 0 {
         return Ok(SyscallReturn::Return(0));
     }
+    if len > usize::MAX - PAGE_SIZE + 1 {
```

Comment:
I thought all occurrences of `usize::MAX - PAGE_SIZE + 1` will be replaced with `isize::MAX`.

Reply:
Replaced